operators/pkg/controller/kibana/driver.go

```diff
@@ -122,9 +122,7 @@ func (d *driver) deploymentParams(kb *kbtype.Kibana) (*DeploymentParams, error)
 	if err != nil {
 		return nil, err
 	}
-	for key := range configSecret.Data {
-		configChecksum.Write(configSecret.Data[key])
-	}
+	configChecksum.Write(configSecret.Data[config.SettingsFilename])
```

Comment:
Why don't we care about the rest of the values in this secret anymore?

Reply:
I want to avoid to recreate the Kibana replicaSet when the operator is upgraded (this implies the `telemetry.yml` has changed and so the `configChecksum`).